desktop/apps/sitemaps/routes.coffee

```diff
@@ -1,21 +1,18 @@
 _ = require 'underscore'
 Articles = require '../../collections/articles'
-Artwork = require '../../models/artwork'
 request = require 'superagent'
 moment = require 'moment'
 async = require 'async'
 PartnerFeaturedCities = require '../../collections/partner_featured_cities'
-{ NODE_ENV, API_URL, POSITRON_URL, FUSION_URL, APP_URL, ARTSY_EDITORIAL_CHANNEL } = require('sharify').data
+{ NODE_ENV, API_URL, POSITRON_URL, APP_URL, ARTSY_EDITORIAL_CHANNEL } = require('sharify').data
 artsyXapp = require 'artsy-xapp'
 PAGE_SIZE = 100
-FUSION_PAGE_SIZE = 5000
 
-epoch = -> moment('2010 9 1', 'YYYY MM DD')
-buckets = _.times moment().diff(epoch(), 'months'), (i) ->
-  {
-    start: epoch().add(i, 'months').format('YYYY-MM-DD')
-    end: epoch().add(i + 1, 'months').format('YYYY-MM-DD')
-  }
+{ SITEMAP_BASE_URL } = require('../../config')
```

Comment:
Assuming this is sensitive -- if not we can add to [sharify](https://github.com/artsy/force/blob/master/desktop/lib/setup_sharify.coffee#L77)